appveyor.yml

```diff
@@ -20,11 +20,29 @@ environment:
 
   matrix:
     - TARGET_ARCH: x86
+      CONDA_PY: 27
       CONDA_INSTALL_LOCN: C:\\Miniconda
 
     - TARGET_ARCH: x64
+      CONDA_PY: 27
       CONDA_INSTALL_LOCN: C:\\Miniconda-x64
 
+    - TARGET_ARCH: x86
+      CONDA_PY: 34
+      CONDA_INSTALL_LOCN: C:\\Miniconda3
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 34
+      CONDA_INSTALL_LOCN: C:\\Miniconda3-x64
+
+    - TARGET_ARCH: x86
+      CONDA_PY: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35
+
+    - TARGET_ARCH: x64
+      CONDA_PY: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
```

Comment:
Note to self: this was rerendered with a conda-smithy that used a cached (and hacked) local index. The rerendering happened quite fast and the matrix is guarantee to be correct since those are the packages available with a `.condarc` configured like:


```yaml
channels:
  - conda-forge
  - defaults
```